h/static/styles/partials-v2/_search.scss

```diff
@@ -49,35 +46,55 @@
   }
 }
 
+.search-result-bucket__header {
+  padding: 10px 10px;
+
+  display: flex;
+  flex-direction: row;
+
+  cursor: pointer;
+  user-select: none;
```

Comment:
I've just worked out that you didn't add this, so please ignore this comment. I reiterate my point from [this comment](https://github.com/hypothesis/h/pull/3790/files#r77322163).